src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Azs.Compute.Admin.psm1

```diff
@@ -24,10 +24,17 @@ if (Test-Path -Path $ClrPath -PathType Container) {
     Get-ChildItem -Path $allDllsPath -File | ForEach-Object { Add-Type -Path $_.FullName -ErrorAction SilentlyContinue }
 }
 
+# Load all CustomObjects
+
 . (Join-Path -Path $PSScriptRoot -ChildPath 'New-ServiceClient.ps1')
 . (Join-Path -Path $PSScriptRoot -ChildPath 'Get-TaskResult.ps1')
 . (Join-Path -Path $PSScriptRoot -ChildPath 'Get-ApplicableFilters.ps1')
 . (Join-Path -Path $PSScriptRoot -ChildPath 'Test-FilteredResult.ps1')
 . (Join-Path -Path $PSScriptRoot -ChildPath 'Get-ArmResourceIdParameterValue.ps1')
+. (Join-Path -Path $PSScriptRoot -ChildPath 'Get-ResourceNameSuffix.ps1')
```

Comment:
dont we need the custom quota, platform image files here

Reply:
No, they are under the ScriptsToProcess in the psd1